lib/Controller/AlbumsController.php

```diff
@@ -136,7 +135,8 @@ private function validFile(File $file, bool $shared): bool {
 	}
 
 	private function isShared(Node $node): bool {
-		return $node->getStorage()->instanceOfStorage(SharedStorage::class);
+		return $node->getStorage()->instanceOfStorage(SharedStorage::class) ||
+			$node->getStorage()->instanceOfStorage(\OCA\GroupFolders\Mount\GroupFolderStorage::class);
```

Comment:
`use OCA\GroupFolders\Mount\GroupFolderStorage` ?

Reply:
No lets not import it with use. As we can't know for sure it is available.
This just expands to the classname.